contrib/k8s/dex-worker.yaml

```diff
@@ -0,0 +1,113 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: dex
+type: Opaque
+data:
+  key-secrets: ZUhoNGVIaDRlSGg0ZUhoNGVIaDRlSGg0ZUhoNGVIaDRlSGg0ZUhoNGVIZz0= # 32 x's base64 encoded twice.
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: dex
+    role: worker
+  name: dex-worker
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: dex
+        role: worker
+    spec:
+      containers:
+          - image: quay.io/coreos/dex
+            name: dex-worker
+            env:
```

Comment:
For that, I was actually thinking that it would be best to be read in directly by the program. Otherwise it's a bit less secret, as the `$(cat key)` will show up in `ps aux` from the host or inside the container, and environment can be viewed with `ps auxe`. The only safe way is probably to read and then delete the secret directly from disk.